renovate.json

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": ["config:base", ":gitSignOff"],
+  "rangeStrategy": "pin",
+  "commitBody": "Signed-off-by: {{{gitAuthor}}}",
```

Comment:
Don't we need to configure the author in this file, too?
See: https://docs.renovatebot.com/configuration-options/#gitauthor

Reply:
I guess when not configuring it, it falls back to some "renovate whatever" being the git author. 
cf. https://github.com/renovatebot/renovate/issues/7273
"If gitAuthor is null (i.e. not configured by the bot admin or within the repo's config) then a call to platform.getGitAuthor() is made. If the result is still null, then we fall back to renovate@whitesourcesoftware.com and log a message that the default gitAuthor is being used."

I guess that would be fine for us?

Reply:
if we extend :gitSignOff, we do not need to have line 4 - thats exactly what :gitSignOff does, isn't it? 
https://docs.renovatebot.com/presets-default/#gitsignoff